src/ansiblelint/rules/risky_octal.md

```diff
@@ -1,6 +1,11 @@
 # risky-octal
 
-This rule checks that octal file permissions either contain a leading zero or are a string value.
+This rule checks that octal file permissions are strings that contain a leading
+zero or are written in symbolic mode, such as `u+rwx` or `u=rw,g=r,o=r`.
+
+This is because using a numeric decimal or octal in YAML can easily lead to
+costly unexpected mistakes. For example, `0644` is interpreted as `420` as
+a decimal.
```

Comment:
```suggestion
Using an octal string in YAML can result in unexpected error. For example, the YAML parser can interpret `0644` as the decimal number `420`.
```

Reply:
@ssbarnea Not sure if this changes meaning too much but I think it needs to be clarified a little more.

Reply:
I used your version to produce something in between as you got a small mistake.